src/main/resources/scripts/cleanup/non_prod_primary_and_replica_data_cleanup.sql

```diff
@@ -0,0 +1,92 @@
+--
+-- RWA-2482 cleanup old tasks from primary task database
+--
+
+CREATE OR REPLACE FUNCTION cft_task_db.task_cleanup_between_dates_primary()
+RETURNS VOID
+LANGUAGE plpgsql
+AS $function$
+DECLARE
+    task_id_var text;
+BEGIN
+    -- Loop through the rows of the tasks table
+    FOR task_id_var IN SELECT task_id FROM cft_task_db.tasks WHERE created >= '<From Date and time>' AND created <= '<To Date and time>' limit 10000 LOOP
```

Comment:
maybe better to pass the from and to date and limits as params into the function?

Reply:
order by time and clear the oldest first

Reply:
Added orderby and added created to and from params